lib/merged_user.coffee

```diff
@@ -0,0 +1,9 @@
+_ = require 'underscore'
+Backbone = require 'backbone'
+DesktopUser = require '../desktop/models/current_user.coffee'
+MobileUser = require '../mobile/models/current_user.coffee'
+
+module.exports = class MergedUser extends Backbone.Model
+
+  _.extend @prototype, DesktopUser.prototype
+  _.extend @prototype, MobileUser.prototype
```

Comment:
Moved this to coffeescript + mixins b/c I think using the `.extend` from Backbone wasn't playing nice with multiple inheritence. Post-merger I'd like to actually merge the two backbone model classes into one—but tricky with it being a moving target rn.